docs/introduction.rst

```diff
@@ -106,12 +106,11 @@ problematic when using libraries.
 :func:`threading.local`. It provides no mechanism of context sharing when a
 child task is spawned. The project looks abandoned.
 
-In the future, asynchronous context storage could be supported natively in the
-Python language. This is discussed in `PEP 550`_ and `PEP 567`_.
+`contextvars`_ is the native solution to manage context-local states starting
+from Python 3.7.
 
 .. _aiohttp: https://aiohttp.readthedocs.io/
 .. _aiolocals: https://docs.atlassian.com/aiolocals/
 .. _aiotask-context: https://github.com/Skyscanner/aiotask-context
 .. _tasklocals: https://github.com/vkryachko/tasklocals
-.. _PEP 550: https://www.python.org/dev/peps/pep-0550/
-.. _PEP 567: https://www.python.org/dev/peps/pep-0567/
+.. _contextvars: https://docs.python.org/3/library/contextvars.html
```

Comment:
@nizox contextvars seems to be promising enough. If I am not mistaken ContextVars will store a strong reference to the value that we are storing and thus might need to be cleaned up manually. Isn't the approach that aiocontext currently uses of storing the additional value as an attribute of Task, better for asyncio than contextvars..?

Reply:
Actually, `contextvars.ContextVar` is only a handle. The value is held in a `contextvars.Context` instance which is bound to a `asyncio.Task`. There is no need to clean it up manually. The [PEP-0567](https://www.python.org/dev/peps/pep-0567/#asyncio) provides details on the asyncio implementation.